contracts/LPFeeManager.sol

```diff
@@ -0,0 +1,161 @@
+import './interfaces/IERC20.sol';
+import '@openzeppelin/contracts/access/Ownable.sol';
+
+/// @title LP fee manager
+/// @author Apeswap.finance
+/// @notice Swap LP token fees collected to different token
+contract LPFeeManager is Ownable {
+    address[] possiblePaths;
+    IApeRouter02 router;
+    IApeFactory factory;
+
+    uint256 public slippageFactor; //divided by 1000
+
+    event ChangedPossiblePaths(address[] _paths);
+    event LiquidityTokensSwapped(address[] _lpTokens, address _outputToken, address _to);
+
+    constructor(
+        address[] memory _possiblePaths,
+        address _router,
+        address _factory,
+        uint256 _slippageFactor
+    ) public {
+        possiblePaths = _possiblePaths;
+        router = IApeRouter02(_router);
+        factory = IApeFactory(_factory);
+        slippageFactor = _slippageFactor;
+    }
+
+    /// @notice Swap LP token fees collected to different token
+    /// @param _lpTokens address list of LP tokens to swap to _outputToken
+    /// @param _outputToken ERC20 token the LP tokens needs to be swapped to. Address(0) if you only want split LP tokens.
+    /// @param _to address the tokens need to be transferred to.
+    function swapLiquidityTokens(
+        address[] memory _lpTokens,
+        address _outputToken,
+        address _to
+    ) public onlyOwner {
+        for (uint256 i = 0; i < _lpTokens.length; i++) {
+            _swapLiquidityToken(_lpTokens[i], _outputToken, _to);
+        }
+        emit LiquidityTokensSwapped(_lpTokens, _outputToken, _to);
+    }
+
+    /// @notice private function. Same as swapLiquidityTokens but for 1 LP token.
+    /// @param _lpToken LP tokens to swap to _outputToken
+    /// @param _outputToken ERC20 token the LP tokens needs to be swapped to. Address(0) if you only want to split LP tokens.
+    /// @param _to address the tokens need to be transferred to.
+    function _swapLiquidityToken(
+        address _lpToken,
+        address _outputToken,
+        address _to
+    ) private {
+        address LPTokensToAddress = _outputToken == address(0) ? _to : address(this);
+
+        IApePair pair = IApePair(_lpToken);
+        uint256 balance = pair.balanceOf(address(this));
+        pair.approve(address(router), balance);
+        router.removeLiquidity(pair.token0(), pair.token1(), balance, 0, 0, LPTokensToAddress, block.timestamp + 600);
+
+        if (_outputToken != address(0)) {
+            _swapTokens(pair.token0(), _outputToken, _to);
+            _swapTokens(pair.token1(), _outputToken, _to);
+        }
+    }
+
+    /// @notice Swapping of tokens to _outputToken
+    /// @param _token token to swap to _outputToken
+    /// @param _outputToken ERC20 token the LP tokens needs to be swapped to
+    /// @param _to address the tokens need to be transferred to.
+    function _swapTokens(
+        address _token,
+        address _outputToken,
+        address _to
+    ) private {
+        if (_token == _outputToken) {
```

Comment:
I think it would be preferable to include a separate `onlyOwner` function for sweeping tokens as we normally because this logic is a bit unexpected. 

```
    /// @notice A public function to sweep accidental ERC20 transfers to this contract. 
    /// @param _tokens Array of ERC20 addresses to sweep
    /// @param _to Address to send tokens to
    function sweepTokens(ERC20[] memory _tokens, address _to) external onlyAdmin {
        for (uint256 index = 0; index < _tokens.length; index++) {
            ERC20 token = _tokens[index];
            require(token != IAZO_INFO.IAZO_TOKEN, "cannot sweep IAZO_TOKEN");
            require(token != IAZO_INFO.BASE_TOKEN, "cannot sweep BASE_TOKEN");
            uint256 balance = token.balanceOf(address(this));
            token.safeTransfer(_to, balance);
            emit SweepWithdraw(_to, token, balance);
        }
    }
```